modules/tinymce/src/themes/silver/main/ts/ui/general/OuterContainer.ts

```diff
@@ -207,7 +207,9 @@ const factory: UiSketcher.CompositeSketchFactory<OuterContainerSketchDetail, Out
         Attribute.remove(element, 'aria-hidden');
       });
 
-      apis.refreshToolbar(comp);
+      setTimeout(() => {
```

Comment:
I think it would be better if we some how call this refresh when the view is being updated. So in the ViewWrapper toggleView call somehow execute the outer container refresh. The toggleView is called from the OuterContainer module so it should have access to that component in there.

Reply:
No wait miss reading this so is it we need to wait for the autoresize to happen then maybe change the Iframe code that waits for events then calls a refresh on the OuterContainer on various events but only on sliding and floating modes maybe it should do so for this mode as well.

Reply:
ok, I maybe have found a different solution

Reply:
I poked around with this couldn't find a better way to solve this I think the real bug is in DockingApis and that it updates and measures things while they are `display: none`. I tried adding a `component.element.dom.offsetParent` nul check in the refresh code in DockingApis and that seems to fix it as well that just ignores the update from happening if the element is or is within a `display: none`. This is wider than just views this will happen if scroll happens while the editor is hidden say in a tab view etc.

Reply:
sadly add `component.element.dom.offsetParent` make a regression for [TINY-9419](https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/src/themes/silver/test/ts/browser/view/ViewTest.ts#L419-L431), so I would stay to the current solution